composer.json

```diff
@@ -20,6 +20,7 @@
   },
   "require": {
     "php": ">=5.3.3",
-    "apache/log4php": "2.3.0"
+    "apache/log4php": "2.3.0",
```

Comment:
Perhaps for v2.0.0 we can get rid of any "default" logger, and just clearly state that the user should add one by him/herself. No biggie, since we're going to do a major version change. And it's not unreasonable to ask for a logger as the only dependency (out of the dev environment, of course).